packages/app-mobile/package.json

```diff
@@ -87,24 +87,25 @@
     "url": "0.11.4"
   },
   "devDependencies": {
-    "@babel/core": "7.24.7",
+    "@babel/core": "7.26.0",
     "@babel/plugin-transform-export-namespace-from": "7.24.7",
-    "@babel/preset-env": "7.24.7",
-    "@babel/runtime": "7.24.7",
+    "@babel/preset-env": "7.25.3",
+    "@babel/runtime": "7.25.0",
     "@joplin/tools": "~3.3",
     "@js-draw/material-icons": "1.29.1",
     "@pmmmwh/react-refresh-webpack-plugin": "^0.5.15",
-    "@react-native/babel-preset": "0.74.86",
-    "@react-native/metro-config": "0.74.87",
+    "@react-native-community/cli": "15.0.1",
+    "@react-native-community/cli-platform-android": "15.0.1",
+    "@react-native-community/cli-platform-ios": "15.0.1",
+    "@react-native/babel-preset": "0.77.2",
+    "@react-native/metro-config": "0.77.2",
+    "@react-native/typescript-config": "0.77.2",
     "@sqlite.org/sqlite-wasm": "3.46.0-build2",
-    "@testing-library/jest-native": "5.4.3",
-    "@testing-library/react-native": "12.3.3",
-    "@tsconfig/react-native": "2.0.2",
+    "@testing-library/react-native": "13.2.0",
```

Comment:
Check whether it's possible to do the `@testing-library/react-native` upgrade separately. This change is causing some of the tests to fail.